P5/Source/Specs/att.datable.xml

```diff
@@ -84,7 +107,7 @@ $Id$
     </attDef>
     <attDef ident="period" usage="opt">
       <desc versionDate="2021-04-19" xml:lang="en">supplies pointers to one or more definitions of 
-        named periods of time (typically <gi>category</gi>s or <gi>calendar</gi>s) within which the datable item is understood to have occurred.</desc>
+        named periods of time (typically <gi>category</gi>s, <gi>date</gi>s or <gi>event</gi>s) within which the datable item is understood to have occurred.</desc>
```

Comment:
We should prefer the Oxford comma. But I think we can do better than the awkwardly pluralized element names. Maybe this, removing the parentheses: 

```xml
supplies pointers to one or more definitions of named periods of time, typically the <gi>category</gi>, <gi>date</gi>, or <gi>event</gi> elements indicating when the datable item is understood to have occurred. 
```

Reply:
I agree completely the Oxford comma should be used. But the rewording to avoid those icky plural element names alters the semantics. (From “occured within” to “when occurred”.)
How about
~~~xml
supplies pointers to one or more definitions of named periods of time,
within which the datable item is understood to have occurred;
typically each named period is encoded as a <gi>category</gi>, 
<gi>date</gi>, or <gi>event</gi>.
~~~
?

Reply:
@sydb point taken about changing the semantics, but please, let's not complicate with weird comma and semicolon punctuation. How about this?

```xml
supplies pointers to one or more definitions 
of named periods of time, typically the <gi>category</gi>, 
<gi>date</gi>, or <gi>event</gi> elements that indicate the timeframe for a datable item.
```

Reply:
^^^ edited one more time to simplify.